src/models/subscription.test.ts

```diff
@@ -197,8 +194,82 @@ describe("Subscription", () => {
 		// TODO: add tests
 	});
 
-	describe.skip("install", () => {
-		// TODO: add tests
+	describe("install", () => {
+		describe("cloud", () => {
+			let cloudSub =
+			beforeEach(async ()=>{
+				cloudSub = await Subscription.install({
+					installationId: GITHUHB_INSTALLATION_ID,
+					host: "http://normal-cloud.atlassian.net",
+					gitHubAppId: undefined,
+					clientKey: "cloud_client_key"
+				});
+			});
+			it("should install subscription", async () => {
+				const found = await Subscription.findByPk(cloudSub.id);
+				expect(found).toEqual(expect.objectContaining({
+					gitHubInstallationId: GITHUHB_INSTALLATION_ID,
+					jiraHost: "http://normal-cloud.atlassian.net",
+					gitHubAppId: null,
+					jiraClientKey: "cloud_client_key"
+				}));
+			});
+			it("should override existing record if found", async () => {
+				//install another payload with same value
+				const cloudSub2 = await Subscription.install({
+					installationId: GITHUHB_INSTALLATION_ID,
```

Comment:
/nit Can you fix this typo too?
```suggestion
					installationId: GITHUB_INSTALLATION_ID,
```

Reply:
nice pickup, fixed.